shuffle/cli/src/shared.rs

```diff
@@ -269,6 +267,58 @@ impl Home {
             )),
         }
     }
+
+    pub fn write_network_into_config(&self) -> Result<()> {
+        let network_config_path = self.shuffle_path.join("Networks.toml");
+        let network_config_string = toml::to_string_pretty(&generate_localhost_network()?)?;
+        fs::write(network_config_path, network_config_string)?;
+        Ok(())
+    }
+}
+
+fn generate_localhost_network() -> Result<Value> {
+    let local_host_value = r#"
+        [localhost]
+        base = 'http://127.0.0.1'
+        json_rpc_port = 8080
+        dev_api_port = 8081
+    "#;
```

Comment:
Let's not write to a .toml file with a string literal. Create a struct and then persist that strongly typed struct to a file to toml::to_string...